### ISSUE TITLE
Animate mode and Inverse kinematics issues

### ISSUE THREAD

**Issue by openanim** [opentoonz/opentoonz#3414]
**Describe the bug**
It seems that the animate mode does not work correctly in the latest versions of openoonz. It has become very difficult to select the correct part of the build in animate mode. On OTX everything seems normal to me until the May 2020 version.

The same goes with reverse kinematics which works in a weird way. We can no longer block the joints by setting pinned center where we want , in frame range neither.

**To Reproduce**
List steps to reproduce bad selection
1. Select skeleton tool and Build a skeleton
2. Use animate mode to try select part of the build in the viewer
3. in most case there are a bad selection of the build section

List steps to reproduce bad behavior pinned center
1. Use skeleton tool in IK mode
2. Try to set a pinned center by clicking on a joint 
3. Nothing happens

**Expected behavior**

Same behaviour as previous versions.

**Comment by artisteacher** [comment]
The pinning is related to #3275.

> Clicking on the joint will no longer change the main pin. Shift clicking sets a temporary pin.
Clicking on the root joint now allows the user to move the skeleton around. 

I've always found IK mode fussy, so I tend to avoid it.

I'm not seeing an obvious difference in selecting with animate mode. If a drawing is just a thin stroke, it can be harder to select since the target area is smaller. If a drawing is behind another, it can be tricky to select too. I don't think that's anything new.

**Comment by openanim** [comment]
Thanks I missed this...however I think there is really an issue because as you can see with the same example, in the current version the selection is wrong besides being difficult.

![bad selection](https://user-images.githubusercontent.com/29790512/88347974-77d6dd00-cd4c-11ea-83b2-a77cee69942e.gif)

In a previous OTX the selection works fine although we have to do it carefully.

![correct selection](https://user-images.githubusercontent.com/29790512/88348042-ace32f80-cd4c-11ea-9952-30f66adb9f32.gif)

**Comment by artisteacher** [comment]
Yeah, I can see what you mean. I'm not having that issue on macOS with a nightly build from June 29th.

**Comment by RodneyBaker** [comment]
Closing as the original reporter has ghosted themself.
Note that there may still be useful information contained in this report and where appropriate that information should be transferred to a new report.
